src/common.lisp

```diff
@@ -6,22 +6,69 @@
 
 (defvar *api-key*)
 
+(defvar *webhook-secret*
+  "The endpoint's secret, used to generate [webhook signatures](https://docs.stripe.com/webhooks/signatures).")
+
 (defun normalize-string (string)
   (substitute #\- #\_ (string-upcase string) :test #'char=))
 
 (defun normalize-json-key (string)
   (u:make-keyword (normalize-string string)))
 
-(defun decode-list (list)
-  (destructuring-bind (&key data has-more &allow-other-keys) list
+(defun decode-list (hash-table)
```

Comment:
Would it be better if we rename this function since we're decoding a hash table now @aadcg?